### PR TITLE
Change registration form fields to match mock. Add checkboxes.

### PR DESCRIPTION
**Pull request by MarkWrightBjss**
Added checkboxes to dynamic form. Some reformatting to make style police happy. Merge to WCS.

### CODE REVIEW

File: src/shared/common/forms/checkbox-group.jsx [hunk 1]
```diff
@@ -5,24 +5,29 @@ class Checkbox extends Component {
...
-            if (currentState.value.includes(targetValue)) {
-                return { value: currentState.value.replace(targetValue, '') };
-            } else {
-                return { value: currentState.value + targetValue };
-            }
+        var selection = this.state.value;
+        if (selection.includes(e.target.value)) {
+            selection = selection.filter(cb => cb !== e.target.value);
+        } else {
+            selection.push(e.target.value);
```
Comment (lewismuirhead) [inline src/shared/common/forms/checkbox-group.jsx]: I think you should use array spread to create a new array here

File: src/shared/common/forms/checkbox-group.jsx [hunk 2]
```diff
@@ -5,24 +5,29 @@ class Checkbox extends Component {
...
 
     handleChange(e) {
-        let targetValue = e.target.value + ',';
-        this.setState((currentState) => {
-            if (currentState.value.includes(targetValue)) {
-                return { value: currentState.value.replace(targetValue, '') };
-            } else {
-                return { value: currentState.value + targetValue };
-            }
+        var selection = this.state.value;
```
Comment (lewismuirhead) [inline src/shared/common/forms/checkbox-group.jsx]: as the setstate depends on current state, use the current state argument from the updater function

File: src/shared/common/forms/checkbox-group.jsx (hunk 1 above)
Comment (MarkWrightBjss) [reply src/shared/common/forms/checkbox-group.jsx]: Implemented recommendation.

File: src/shared/common/forms/checkbox-group.jsx (hunk 2 above)
Comment (MarkWrightBjss) [reply src/shared/common/forms/checkbox-group.jsx]: Implemented as suggested.

**Pull request merged**